build/Jamfile.v2

```diff
@@ -15,12 +17,6 @@ import feature ;
 import configure ;
 import atomic-arch-config ;
 
-local here = [ modules.binding $(__name__) ] ;
-
-project.push-current [ project.current ] ;
-project.load [ path.join [ path.make $(here:D) ] ../../config/checks/architecture ] ;
-project.pop-current ;
-
 lib synchronization ;
```

Comment:
@grafikrobot BTW, should this be `searched-lib`? I noticed, you changed `lib` to `searched-lib` in other libraries. What's the difference?